spec/migrations/20170710132935_custom_button_to_classes_spec.rb

```diff
@@ -0,0 +1,61 @@
+require_migration
+
+describe CustomButtonToClasses do
+  let(:custom_button_stub) { migration_stub(:CustomButton) }
+  let(:miq_set_stub) { migration_stub(:MiqSet) }
+
+  shared_examples 'migrate up' do
+    it 'sets the image class and color' do
+      migrate
+      custom_button.reload
+
+      expect(params).to have_key(:button_icon)
+      expect(params).not_to have_key(:button_image)
+    end
+  end
+
+  shared_examples 'migrate down' do
+    it 'reverts the image to a number' do
+      migrate
+      custom_button.reload
+
+      expect(params).to have_key(:button_image)
+      expect(params).not_to have_key(:button_icon)
+      expect(params).not_to have_key(:button_color)
+    end
+  end
+
+  context 'CustomButton' do
+    let(:params) { custom_button.options }
+    let!(:custom_button) { migration_stub(:CustomButton).create!(create_params) }
```

Comment:
you probably intended to use `custom_button_stub` which is already defined for this stub, right?